requirements.txt

```diff
@@ -0,0 +1,3 @@
+pyaml
+pysha3
+jsonschema
```

Comment:
Can you limit/narrow this to at least the major versions of the libraries, so ``pyaml>=17.0,<18.0``, ``pysha3>=1.0,<2.0`` and ``jsonschema>=2.6,<3.0``, otherwise chances are high that things will break on the next major update of a library.

Wonder if one should even stick to the minor versions, but maybe for these generic types of libraries that's ok (though my over-time experience is, that stuff like this breaks rather more often than expected, but choose as you think is appropriate).